features/identity-event/org.wso2.carbon.identity.event.server.feature/resources/org.wso2.carbon.identity.event.server.feature.default.json

```diff
@@ -119,5 +119,11 @@
   "identity_mgt.events.schemes.OIDCLogoutEventHandler.subscriptions": [
     "SESSION_TERMINATE"
   ],
+  "identity_mgt.events.schemes.'default.notification.sender'.module_index": "20",
```

Comment:
Need to update the j2 file to start the custom module indexes from 21 now

Reply:
This value is not hardcoded in the j2. The value is updated dynamically.